### DISCUSSION TITLE
表单校验触发validator时如何获取上一次的错误信息？getFieldError返回空

### DISCUSSION THREAD

**Discussion by h1orz** [ant-design/ant-design#43268] [Q&A] [answered]
如果我希望在Input失焦时校验值的合法性，用onChange校验空值，用onBlur校验合法性，且不希望ErrorList用多个错误，应该怎么设计交互？头有点晕...
示例：https://codesandbox.io/s/ji-ben-shi-yong-antd-5-6-3-forked-dg2kxc?file=/demo.tsx
先点Submit按钮，再聚焦Input，然后失焦。
失焦的时候，我希望“请填写”的错误信息保持柱，应该怎么搞啊？

说明：
1. onBlur判断合法性的原因是每次校验性能消耗比较大，且用户没输入完就提示错误也不友好，故此希望在失焦时校验合法性。
2. onChange校验空值的可以在用户删除所有输入值时，马上提示必填

**Comment by linxianxi** [comment, marked as answer]
添加这两处即可
<img width="649" alt="image" src="https://github.com/ant-design/ant-design/assets/47104575/47bc9fdc-1b15-4177-9965-6cc34995ad7f">

**Reply by h1orz** [reply]
validateFirst 解决大问题，多谢~
